src/day13.scala

```diff
@@ -0,0 +1,66 @@
+// using scala 3.0.2
+
+package day13
+
+import scala.util.Using
+import scala.io.Source
+
+@main def part1(): Unit =
+  val answer = part1(readInput())
+  println(s"The answer is: $answer")
+
+@main def part2(): Unit =
+  val answer = part2(readInput())
+  println(s"The answer is:\n$answer")
+
+def readInput(): String =
+  Using.resource(Source.fromFile("input/day13"))(_.mkString)
+
+def part1(input: String): Int =
+  val (dots, folds) = parseInstructions(input)
+  dots.map(folds(0).apply).size
+
+def part2(input: String): String =
+  val (dots, folds) = parseInstructions(input)
+  val foldedDots = folds.foldLeft(dots)((dots, fold) => dots.map(fold.apply))
```

Comment:
`dots` is shadowed in the `foldLeft`. You might consider another variable name

Reply:
Unfortunately I cannot find a better name.

Reply:
`foldedDotsAcc`? Not sure it is better :laughing: